include/bsp/xi_bsp_io_fs.h

```diff
@@ -7,10 +7,27 @@
 #ifndef __XI_BSP_IO_FS_H__
 #define __XI_BSP_IO_FS_H__
 
+/**
+ * @file xi_bsp_io.fs.h
+ * @brief Xively Client's Board Support Platform (BSP) for File Access
```

Comment:
I've been referring to our BSPs as "Board Support Package" instead of "Platform". Is this our preferred term?